### PR TITLE
[WIP] [OPS-964] Automatically upload to launchpad

### PR DESCRIPTION
**Pull request by balsoft**
## Description

Reduce manual labour of @rvem by uploading to launchpad from CI.

## Related issue(s)

None

#### Related changes (conditional)

- [ ] I checked whether I should update the [README](../tree/master/README.md)

#### Stylistic guide (mandatory)

- [x] My commits comply with [the policy used in Serokell](https://www.notion.so/serokell/Where-and-how-to-commit-your-work-58f8973a4b3142c8abbd2e6fd5b3a08e).

### CODE REVIEW

**Comment by kirelagin** [comment]
@balsoft reuse check fails

File: scripts/dput.nix [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# SPDX-FileCopyrightText: 2019 TQ Tezos <https://tqtezos.com/>
+#
+# SPDX-License-Identifier: MPL-2.0
+
+{ writeShellScriptBin, fetchbzr, python2 }:
+let
+  src = fetchbzr {
+    url = "lp:~alexlist/dput/trunk";
+    rev = "1s";
```
Comment (rvem) [inline scripts/dput.nix:9]:
```suggestion
    rev = "1";
```

File: scripts/launchpad.sh [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+trap finish EXIT
+
+nix-build -A debSource -o source-packages \
+  --arg builderInfo "\"Serokell <hi@serokell.io>\"" \
+  --arg timestamp "$(date +\"%Y%m%d%H%M\")" --arg date "\"$(date -R)\""
+
+cp source-packages/* "$TEMPDIR"
+chmod 700 -R "$TEMPDIR"
+debsign "$TEMPDIR"/*.changes
+dput -c ppa:serokell/tezos "$TEMPDIR"/*.changes
```
Comment (rvem) [inline scripts/launchpad.sh:23]:
```suggestion
for file in "$(ls "$TEMPDIR"/*.changes)"; do
    dput -c ppa:serokell/tezos "$file"
done
```

Comment (rvem) [reply scripts/launchpad.sh:23]: Also, why `-c` is required for `dput` here? :thinking:

**Comment by gromakovsky** [comment]
Most likely we won't need anything from this PR because we are changing the way of packaging (and we've already changed the way of building).

**Pull request closed without merging**